### ISSUE TITLE
Chokes on `const enum`

### ISSUE THREAD

**Issue by anurbol** [swc-project/swc#516]
SWC Parser panics when sees this:
```
const enum Foo {}
```
The code is perfectly valid: https://www.typescriptlang.org/docs/handbook/enums.html#const-enums

The error:

```
panicked at 'internal error: entered unreachable code: Current token is `None`. Parser should not call bump()without knowing current token', C:\Users\nurbol\.cargo\registry\src\github.com-1ecc6299db9ec823\swc_ecma_parser-0.13.0\src\parser\input.rs:208:21

Stack:

Error
    at module.exports.__wbg_new_59cb74e423758ede (..../parser_js_ts.js:135:17)
    at console_error_panic_hook::hook::h8d774dc82a24253f (wasm-function[527]:0x20d84e)
    at core::ops::function::Fn::call::h0c0944b36a2e81a1 (wasm-function[2311]:0x2551f9)
    at std::panicking::rust_panic_with_hook::hea54eab3988c00c4 (wasm-function[923]:0x2356c5)
    at std::panicking::continue_panic_fmt::h2ff8f439532e34d3 (wasm-function[1411]:0x24a235)
    at std::panicking::begin_panic_fmt::ha0c6adbba1980c83 (wasm-function[1661]:0x250623)
    at swc_ecma_parser::parser::stmt::<impl swc_ecma_parser::parser::Parser<I>>::parse_stmt_internal::hdf2c271664424e2e (wasm-function[13]:0x1b90)
    at swc_ecma_parser::parser::stmt::<impl swc_ecma_parser::parser::Parser<I>>::parse_block_body::h540c02ce05d65079 (wasm-function[85]:0x13f575)
    at swc_ecma_parser::parser::Parser<I>::parse_typescript_module::h8135ec638a616ab3 (wasm-function[217]:0x1b71d4)
    at scoped_tls::ScopedKey<T>::set::h9f084517f40b7d26 (wasm-function[140]:0x18128f)

```

**Comment by kdy1** [comment]
I published swc_ecma_parser v0.13.1

**Comment by anurbol** [comment]
@kdy1 It works! Thank you!!
